pkg/operator/operator/validations.go

```diff
@@ -744,3 +789,54 @@ func getValidationK8sResources() ([]kunstructured.Unstructured, *kresource.Quant
 
 	return virtualServices, maxMem, err
 }
+
+func validateDockerImagePath(image string) error {
+	if _, err := cr.ValidateImageVersion(image); err != nil {
```

Comment:
I'm considering short-circuiting the check if the image is one of the defaults, what do you think? We could have a `strset` which includes the defaults. It's a little bit of extra state we'd have to keep consistent, but that would be the most common code path through here, so it might be nice to make that faster.

Reply:
I like this suggestion. It can speed up the validation process when default paths are used.